packages/nameguard-sdk/src/index.test.ts

```diff
@@ -64,6 +64,15 @@ describe("NameGuard", () => {
     expect(data.display_name).toBe("nick.eth");
   });
 
+  it("should analyze a primary name and not include the nameguard_result", async () => {
+    const data = await nameguard.getSecurePrimaryName(
+      "0xb8c2C29ee19D8307cb7255e1Cd9CbDE883A267d5",
+      { computeNameGuardReport: false },
+    );
+
+    expect(data.display_name).toBe("nick.eth");
```

Comment:
We should add an additional `expect` here to verify no nameguard_result provided.